common/witness.go

```diff
@@ -0,0 +1,39 @@
+package common
+
+import "github.com/nspcc-dev/neo-go/pkg/interop/runtime"
+
+var (
+	// ErrAlphabetWitnessFailed appears when method must be
+	// called by Alphabet but was not.
+	ErrAlphabetWitnessFailed = "alphabet witness check failed"
+	// ErrOwnerWitnessFailed appears when method must be called
+	// by owner of some assets but was not.
+	ErrOwnerWitnessFailed = "owner witness check failed"
+	// ErrWitnessFailed appears when method must be called
+	// using certain public key but was not.
+	ErrWitnessFailed = "witness check failed"
+)
+
+// PanicOnAlphabetWitnessFail checks witness of the passed caller.
+// Panics with ErrAlphabetWitnessFailed message on fail.
+func PanicOnAlphabetWitnessFail(caller []byte) {
```

Comment:
`common.CheckAlphabetWitness` is a better choice IMO, same for other functions.

Reply:
agree, renamed